_episodes/012-pietro-pascal-santiago.md

```diff
@@ -0,0 +1,86 @@
+#reddit: (leave blank on initial publish, amend with link and uncomment this line after Reddit thread has been posted)
+
+# https://github.com/jekyll/jekyll/issues/7744
+layout: episode
+---
+
+Another trio of interviews from RustFest 2019: [Pietro Albini](https://twitter.com/pietroalbini) on Crater and the Rust Infrastructure Team; [Pascal Hertleif](https://twitter.com/killercup) on the Rust Developer Tools Team; and [Santiago Pastorino](https://twitter.com/spastorino) on the [Rust Latam](https://rustlatam.org/) conference in Latin America.
+
+<!--
+The episode introduction goes here.
+The first paragraph should ideally be short, and is used in various
+places as a "short description" for the episode. Any subsequent
+paragraphs show up as "expanded description".
+-->
+
+### Contributing to Rustacean Station
+
+<!-- You can probably leave this as-is -->
+
+Rustacean Station is a community project; get in touch with us if you'd like to suggest an idea for an episode or offer your services as a host or audio editor!
+
+ - Twitter: [@rustaceanfm](https://twitter.com/rustaceanfm)
+ - Discord: [Rustacean Station](https://discord.gg/cHc3Gyc)
+ - Github: [@rustacean-station](https://github.com/rustacean-station/)
+ - Email: [hello@rustacean-station.org](mailto:hello@rustacean-station.org)
+
+### Timestamps & referenced resources
+
+#### [@00:00] Part 1: Crater & Rust Release Infrastructure w/ Pietro Albini
+
+- [@01:01] - What is your role in the Rust project?
+- [@01:46] - What lessons did the infrastructure team learn from the Rust 2018 release?
+- [@03:29] - How do you feel about potential future Rust editions in 2021 or beyond?
+- [@06:26] - Do you think Rust's regular release cycle too fast or too slow?
+- [@08:56] - How does Crater guard against language regressions, and what things doesn't it catch?
+- [@11:12] - How has Crater scaled as the ecosystem has grown, and is it at risk of becoming infeasible to run?
+- [@16:17] - How can someone get involved with the Infrastructure Team?
+
+#### [@17:25] Part 2: Developer Tools w/ Pascal Hertleif
+
+- [@18:23] - What is the Developer Tools Team?
+- [@19:39] - What tools is the Developer Tools Team responsible for, and what purposes do they serve?
+- [@24:46] - Which tools in particular would you like to draw attention to?
+- [@26:19] - How does rust-analyzer compare to RLS?
+- [@29:42] - How does the Developer Tools Team coordinate?
+- [@32:00] - How was your experience at RustFest this year?
+
+#### [@36:21] Part 3: Rust Latam w/ Santiago Pastorino
+
+- [@36:46] - What is Rust Latam?
+- [@37:42] - What inspired you to start a Rust conference in Latin America?
+- [@39:06] - How big is Rust Latam?
+- [@40:15] - What is interest in Rust like in Latin America?
+- [@42:42] - What is the broader software industry like in Latin America?
+- [@44:59] - What's next for Rust Latam?
+- [@45:42] - How did you get into Rust?
+- [@50:17] - What venues are there for Spanish or Portuguese-speaking Rust users?
```

Comment:
Relevant links?

Reply:
I didn't actually get any links, mostly it's just "yeah they exist and they're in telegram"